source/shared/net/desktop-wallet/accounts-desktop.rst

```diff
@@ -7,35 +7,43 @@ Use the Desktop Wallet
 
 This topic describes the information and the tasks that are available in the Desktop Wallet. To learn more about creating accounts, see :ref:`Create an account in the Desktop Wallet <create-account>`.
 
--  To view information about an account, go to **Accounts** and select the relevant account.
+To view information about an account, go to **Accounts** and select the relevant account. At the top of the window you see the following:
+
+-  **Wallet total** shows the total amount of CCDs in all accounts in this wallet.
+
+-  **At disposal** shows the amount of CCDs you have available for transactions in all accounts in this wallet. This includes shielded balances but not staked amounts.
+
+-  **Stake** shows the amount of CCDs staked in all accounts for this wallet.
 
 The most common tasks on an account are:
 
 -  **Send**: Send CCD to a recipient in your address book. See :ref:`Create CCD transfer in the Desktop Wallet <send-ccd-wallets>`.
 
 -  **Shield**: Transfer CCD from your public balance to your shielded balance. See :ref:`Shield CCD on an account <shield-ccd-wallets>`.
 
-In addition, you can see the following information about the selected account:
+-  **Receive**: Select this to display the account address. Select the icon to the right to copy the address if you want to share it with other participants in the network or let them scan the QR code.
```

Comment:
We actually have a guard/ledger connection that prompts the user to also display the address on the ledger.
(But that is not very relevant for this PR :upside_down_face: )